clang/tools/offload-arch/CMakeLists.txt

```diff
@@ -1,8 +1,16 @@
 set(LLVM_LINK_COMPONENTS Support)
 
-add_clang_tool(offload-arch OffloadArch.cpp NVPTXArch.cpp AMDGPUArchByKFD.cpp AMDGPUArchByHIP.cpp)
+add_clang_tool(offload-arch OffloadArch.cpp NVPTXArch.cpp AMDGPUArchByKFD.cpp
+               AMDGPUArchByHIP.cpp LevelZeroArch.cpp)
+
+find_path(OFFLOAD_ARCH_LEVEL_ZERO_INCLUDE_DIR NAMES level_zero/ze_api.h)
```

Comment:
if we're requriing the headers we might want to do a full system search, see what we do in our fork [here](https://github.com/intel/llvm/blob/sycl/unified-runtime/cmake/FetchLevelZero.cmake#L16)